pkg/objstore/objtesting/acceptance_e2e_test.go

```diff
@@ -42,6 +42,11 @@ func TestObjStore_AcceptanceTest_e2e(t *testing.T) {
 		testutil.Ok(t, err)
 		testutil.Equals(t, "@test-data@", string(content))
 
+		// Check if we can get the correct size.
```

Comment:
Can we check if ObjectSize for non existing block properly errors? Also if we can find assert NotFound on such operation if block is missing? The latter is not critical, but if we cannot assume so, we should document that as a method comment (:

Reply:
Added a test for this before the initial upload and cleaned up the functions so that `IsObjNotFoundErr(error)` would be properly triggered in such cases.